apps/thehive/analyzers/Vast/VAST-Search.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "VAST-Search",
```

Comment:
We usually talk about queries, not about search, but YMMV. @mavam, maybe something for you to also take a look at.

Reply:
This is the name of the analyzer which will be shown in TheHive/Cortex and it basically searches/queries Vast server. We can decide on another name.

Reply:
These kinda questions are usually well suited for asking in Slack, e.g., just ask "How should we name the TheHive/Cortex analyzer for VAST?"

Reply:
In the Slack discussion, general feedback was using "VAST-Search" or "VAST-Query" for the time being. I'm letting this stay for this iteration. In the next iterations, we can get feedback from community and make changes. Kindly let me know if further discussion is needed here.